collectors/python.d.plugin/mysql/mysql.chart.py

```diff
@@ -612,10 +660,40 @@ def _get_data(self):
             else:
                 self.queries.pop('slave_status')
 
+        if 'user_statistics' in raw_data:
```

Comment:
please add a comment to undertsand what is in `raw_data['user_statistics']`

<details>
<summary>this, for example</summary>

```python
(
    (
        ('netdata', 42L, 0L, 1264L, 3.111252999999968, 2.968510299999994, 110267L, 19741424L, 0L, 0L, 1265L, 0L,
        0L, 0L, 3L, 0L, 1301L, 0L, 0L, 7633L, 0L, 83L, 44L, 0L, 0L),
        ('root', 60L, 0L, 184L, 0.22856499999999966, 0.1601419999999998, 11605L, 1516513L, 0L, 9L, 220L, 0L, 2L, 1L,
        6L, 4L,127L, 0L, 0L, 45L, 0L, 45L, 0L, 0L, 0L)
     ),
 (
     ('User', 253, 9, 128, 128, 0, 0),
     ('Total_connections', 3, 2, 11, 11, 0, 0),
     ('Concurrent_connections', 3, 1, 11, 11, 0, 0),
     ('Connected_time', 3, 4, 11, 11, 0, 0),
     ('Busy_time', 5, 21, 21, 21, 31, 0),
     ('Cpu_time', 5, 18, 21, 21, 31, 0),
     ('Bytes_received', 8, 6, 21, 21, 0, 0),
     ('Bytes_sent', 8, 8, 21, 21, 0, 0),
     ('Binlog_bytes_written', 8, 1, 21, 21, 0, 0),
     ('Rows_read', 8, 1, 21, 21, 0, 0),
     ('Rows_sent', 8, 4, 21, 21, 0, 0),
     ('Rows_deleted', 8, 1, 21, 21, 0, 0),
     ('Rows_inserted', 8, 1, 21, 21, 0, 0),
     ('Rows_updated', 8, 1, 21, 21, 0, 0),
     ('Select_commands', 8, 1, 21, 21, 0, 0),
     ('Update_commands', 8, 1, 21, 21, 0, 0),
     ('Other_commands', 8, 4, 21, 21, 0, 0),
     ('Commit_transactions', 8, 1, 21, 21, 0, 0),
     ('Rollback_transactions', 8, 1, 21, 21, 0, 0),
     ('Denied_connections', 8, 4, 21, 21, 0, 0),
     ('Lost_connections', 8, 1, 21, 21, 0, 0),
     ('Access_denied', 8, 2, 21, 21, 0, 0),
     ('Empty_queries', 8, 2, 21, 21, 0, 0),
     ('Total_ssl_connections', 8, 1, 21, 21, 0, 0),
     ('Max_statement_time_exceeded', 8, 1, 21, 21, 0, 0)),
)
```
</details>

___

i think better to move the whole `user_statistics` to a separate method

Reply:
I hope I made the changes correctly :)